src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -229,28 +229,34 @@ protected int doWork() {
             outputSummaryMetricsFile = new File(OUTPUT + FINGERPRINT_SUMMARY_FILE_SUFFIX);
         }
 
-        IOUtil.assertFileIsReadable(INPUT);
```

Comment:
It would be nice to have this method only check if isn't a URL, or if it is, at least to try to open it!

Reply:
See line 328 below.

Reply:
I don't see anything at 328.  I was talking about changing ` IOUtil.assertFileIsReadable` method, since presumably you'd need to do the same for any other tool in picard to support GCS

Reply:
well...that's another htsjdk change, which I hesitate to make unless there's an obvious need...so once we see several Picard tools making use of the Path interface, we can figure out how we'd like htsjdk to provide that....